tests/stress/test_stress_concurrent.py

```
@@ -0,0 +1,608 @@
+        metrics = StressTestMetrics()
+
+        for concurrency in concurrent_levels:
+            print(f"\nTesting {concurrency} concurrent searches...")
+
+            async def perform_search(query_id: int, query: str, current_concurrency: int = concurrency):
+                start_time = time.time()
+                try:
+                    result = await search_support_docs(ctx=stress_test_context, query=query, limit=10)
+
+                    duration = time.time() - start_time
+                    import json
```

Pylint (3.3.7)

[convention] Import outside toplevel (json)

(C0415)